torchgeo/datasets/deepglobelandcover.py

```diff
@@ -0,0 +1,264 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""DeepGlobe Land Cover Classification Challenge dataset."""
+
+import os
+from typing import Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from matplotlib.figure import Figure
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import (
+    check_integrity,
+    draw_semantic_segmentation_masks,
+    extract_archive,
+    rgb_to_mask,
+)
+
+
+class DeepGlobeLandCover(VisionDataset):
+    """DeepGlobe Land Cover Classification Challenge dataset.
+
+    The `DeepGlobe Land Cover Classification Challenge <https://competitions.codalab.org/competitions/18468/>` dataset 
```

Comment:
```suggestion
    The `DeepGlobe Land Cover Classification Challenge <https://competitions.codalab.org/competitions/18468>`_ dataset 
```
Two things here, the URL doesn't work with a trailing slash, and rST requires a trailing underscore to turn this into a URL (confusing, I know).

Reply:
This helps, thanks!